jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ResourceHandler.java

```diff
@@ -294,6 +304,22 @@ public void setBaseResourceAsString(String base)
         setBaseResource(base == null ? null : ResourceFactory.of(this).newResource(base));
     }
 
+    /**
+     * @param byteBufferSize The size of the byte buffers used to serve static resources.
+     */
+    public void setByteBufferSize(int byteBufferSize)
+    {
+        _byteBufferSize = byteBufferSize;
+    }
+
+    /**
+     * @param useDirectByteBuffers whether to use direct byte buffers to serve static resources.
+     */
+    public void setUseDirectByteBuffers(boolean useDirectByteBuffers)
+    {
+        _useDirectByteBuffers = useDirectByteBuffers;
+    }
```

Comment:
Remove these and just use an optional constructor injected Sized pool.

Reply:
Done.